.github/workflows/api_docs.yml

```
@@ -0,0 +1,66 @@
+name: Sync docs with Readme
+
+on:
+  pull_request:
+    paths:
+      - "docs/pydoc/**"
+  push:
+    branches:
+      - main
+
+env:
+  HATCH_VERSION: "1.9.3"
+  PYTHON_VERSION: "3.10"
+
+jobs:
+  get-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.version_finder.outputs.versions }}
+    steps:
+      - name: Get Haystack Docs versions
+        id: version_finder
+        run: |
+          curl -s "https://dash.readme.com/api/v1/version" --header "authorization: Basic ${{ secrets.README_API_KEY }}" > out
+          VERSIONS=$(jq -c '[ .[] | select(.version | startswith("2.")) | .version ] | .[-2:]' out)
```

I would keep the same behaviour we have for the integrations and update all the `2.x` versions.

```suggestion
          VERSIONS=$(jq '[ .[] | select(.version | startswith("2."))| .version ]' out)
```